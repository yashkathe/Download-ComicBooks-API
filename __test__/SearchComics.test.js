const { getComicsThroughSearch } = require('../index');


test("Search Comics", async () => {
    expect(await getComicsThroughSearch('Batman', 1)).toMatchSnapshot();
});

