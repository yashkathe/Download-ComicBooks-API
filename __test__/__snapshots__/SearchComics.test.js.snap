// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Comics 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Fortress-6-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Having finally located the Fortress of Solitude, Batman and his hastily assembled allies quickly discover that it is as deadly as it is awe-inspiring. With an alien force holding the world hostage while the heroes hunt for Superman, can the Dark Knight find a weapon to repel the invaders? Or will his ragtag team succumb to dysfunction and betrayal before they can save the world?",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/WhVx8udqlOYSEmAklSpe9PeBsA7neMApcX23f+gVg98WcfJSok8hG0AKhaKxgwlRPv0QULu5XP2a5ZGDdHBtV+0Ukw0OSxf8RJot09y8RyBXci76lwwOzGqtClzTNT2HAM1b1N7yGBgLW7AcNcdj6HDRld++x9rt1F0++60LOtE+oeiZ9DYRdStvjHzbi6nL:AMCWbbIYbmfcJyjbfSnKAQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/+Z7xlv8kbF/9d80ZFVnFjzQHcg2IRLR4/Qg6NLq6Qf9D+3BWn98jHc0/vsd4z9whaLYtJ1dClyM/cTgptQtyQIPxwWBRGSN7EcLSlCJjFwuebm1GJRvVCMBas0EoJusWxhMqNptawstZa0kwBetEC29l4FWW2YXJqeKeJ9Dzdn6qIx24zWKTun85thrLiKEn:eWqRt2S2uq95YVDDax9+gw==",
      "MEGA": "https://getcomics.info/links.php/KgbF3JHWGBo9juJ1zHXiIKiVkIF5nxSlnSwohw2NnTInTzQxHrbjxRFPI0X+5SAxclK5aUub92MQ27AE9oWCf+JbQ9yRjT5wKWDWuoWZCK0=:a297Hb8OCIOHBkaSIx8/cw==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-fortress-2022/6",
      "UFILE": "https://getcomics.ufile.io/kjc76wwz",
      "ZIPPYSHARE": "https://www116.zippyshare.com/v/syvwUYk7/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "50 MB",
      "Year": "2022",
    },
    "title": "Batman – Fortress #6 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Beyond-the-White-Knight-6-2022.jpg?fit=400%2C615&ssl=1",
    "description": "After years of rivalry, it’s the showdown you’ve been waiting for…Jason Todd versus Dick Grayson! Who’s the best Robin? Find out this issue! Plus, with the help of an unexpected new ally, Bruce Wayne aims to take Derek Powers down for good. But will the malevolent millionaire’s newfound powers prove to be too much for the former Batman?",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/7gcE0+eM+tS2qD63IZSxP79PrliMWpJCQmdylQbOmh7/OKhnkRd0PbEqjLvZn0Lg0J865T59YAfixZ0fH+cqG9NlqmciYSuz6/urrmaLh8Vv4S7+RIjfP6R/kqtVDg/QeMNZgzEWpwPBgwkJacCgMvOCwdO0cnmrSleHMSYhRcrlBmkt3OuA9q7xcb2Q9OFMFxH+7cuN2+UdykqfBFiSnw==:DRsvq0L7roHLJWdTmdkTUA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/h4K17+XU2vRPQutuVP4+n4Boq8V8kAUxw0z+MBPhHuec5/eDnTOBzmEkm8aNScB8TK23grAfxM0PiTTSey0WBzfhA6j759w8zjQ3P4A9Kp/3oSBOqtJf8QknpT6+troMSiXYPslXPRj+lYOAMwBoNDFZXw2No0h+UdfHQudNRgDrjIk2FDoi/ysSCLp3eNxkTwX4QB/otf8sMdelXRMsBg==:NOhFplwxySp/bmzh98o3aA==",
      "MEGA": "https://getcomics.info/links.php/p0cz7pGRAjvnS1zgWLUxEx5WKaf24+jCHVEasKdpfvAHBTUHXFIphBw/6G9d2BKMzdP/1mDc1BVpNpx1SoQ1L20lMFOyyeltyA+oGoITOpU=:zuc+KvysoN0x0AN0oSv0yw==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-beyond-the-white-knight-2022/6",
      "UFILE": "https://getcomics.ufile.io/nqi3443p",
      "ZIPPYSHARE": "https://www29.zippyshare.com/v/1e33K5C5/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "64 MB",
      "Year": "2022",
    },
    "title": "Batman – Beyond the White Knight #6 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Gotham-Knights-Gilded-City-1-2022.jpg?fit=400%2C615&ssl=1",
    "description": "A mysterious virus has infected Gotham City–turning its victims into rabid, yellow-irised maniacs driven to looting, theft, and bursts of anger. Batman and his Gotham Knights struggle to keep this strange virus contained while investigating its origins. But this is not the first time this unusual illness has overtaken the city. Take a trip to Gotham in the mid-1800s and meet the city’s first masked vigilante–the mysterious hero known only as the Runaway–as they, too, find themselves mired in the mystery of this infection! This thrilling and horrific tale, split between modern-day Gotham and the Gotham of 1847, leads directly into the upcoming video game Gotham Knights!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/YF21i80Ce1ssn7zuS09do6VbxL2BvHbBSQovdP0GUrVQr55/gSi3s4npZ9PKYV5cup63U/BiQ1WbSKEbpMWWjYj+cpGahXXvkuPTxzKnycPBX8m5Wf2fdIMXC7FTrLHLP0udvRcVdvXWG91L5r6hyx8Ie26EZz76vHY+kYmdPQL98bltR+11CNQm6IVDZPO4UWSWISFMw+VIyNKKGrduslfjg1YCj4l/ED605mykbDnvp4mYnLid3HxZ6sA7MZ8F:yNKzWNvfzs4kccNeMSE7Ow==",
      "MEDIAFIRE": "https://getcomics.info/links.php/nINTQjERSi96uNi2HtsLzxs9VSUxammlxCpD4EpNOhTh1ZH9011mbeppbVOfQwQqLrF7j8XF/vCQCVHQEKnAByFZxgtP4mh3J4ygGmm1Lhhs3/PafE7I70CZEJ7kpCq68p5eiDwsWcCXfCb1bYj7gUnmOsR/SNjJz9UjoNg8wIAYxDWN029cjtDd0Xj71B6ZZ2bk8DEnV+R06S6FMVeb/Hc4uhnTHK5/p7p71IVlOq9CooJrI9sW71V6vvYU+8m9:CY9AYsdPpIUCXvW45PEetw==",
      "MEGA": "https://getcomics.info/links.php/ixLuH+GegnGgKh42ncg5rv4Ss9PYzzjmcCW0NsY8OsiXAngCbQtxpf/5rEqKa+iVpkHjp691l5E7Fpo59qNsZhFLDraRR7y9E+X475es6Kc=:t05H79SOs5QSdr3wkSN4yg==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-gotham-knights-gilded-city-2022/1",
      "UFILE": "https://getcomics.ufile.io/3bngtlpx",
      "ZIPPYSHARE": "https://www119.zippyshare.com/v/Scl0hSya/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "116 MB",
      "Year": "2022",
    },
    "title": "Batman – Gotham Knights – Gilded City #1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Gotham-After-Midnight-2022-Fan-Made-TPB.jpg?fit=400%2C615&ssl=1",
    "description": "Trick-or-treating in Gotham City can be terrifying and tragic. Tonight, two people are invited to a party where they will be challenged to surviving the night in the most horrifying haunted house imaginable. Collecting the 12-issue series from Steve Niles and Kelley Jones!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/CYJ7cfm+xuhSss+kvucvKhgt1jNaEI8G5pgnAh7IpwW8V+L4oXottp66JlYi5vN1fLrB1utmGUvhctrtSkdiIfLBZazYaGGFjSM8GKexHrqGCS55ZG0jxDVK/O/Pw2KNQt7D8lW15xQI2xFw9fqd8g==:Uefvl321BC8SmFr22u5fSQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/Z3NjfHgwan+U6v+9/GK+bziyG0CeEjTeV6JKO0lHMLMj0njNNnsEImI13SOfcm0mqhcOL71V5ekNcO1jvoEPoOAfa+rydpYWskbEVm7gNU6rJb+tbeXxDjIWiwT0E4aurOv+Qx0dAEUJTNV5dk7upQ==:F5TVlQd+aY8fTx0GZ2Twng==",
      "MEGA": "https://getcomics.info/links.php/tBzqeR7ZBbW2d0ejWGBeaLbXCsLB2c91yCyB66XGehXLDtRKAzgkszK2CIuR6DK9u+U2HMxmeoWbaMX5utUXKzKrnMUSGrzBN/6XpFg+MC4=:8fTUcWDvlj7GUbg3tBzEzQ==",
      "UFILE": "https://getcomics.ufile.io/lufzc79p",
      "ZIPPYSHARE": "https://www110.zippyshare.com/v/CAqGGKFy/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "352 MB",
      "Year": "2022",
    },
    "title": "Batman – Gotham After Midnight (2022) (Fan-Made TPB)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/I-Am-Batman-14-2022.jpg?fit=400%2C615&ssl=1",
    "description": "The mystery of a pair of brutal murders is solved, a hero is born, and a mantle is passed. The conclusion of the intense “She Got Next” saga will reverberate from New York all the way back to Gotham City as it sets up the next chapters for both Jace Fox and Renee Montoya.",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/gyINrg9yW8SiPcddJlk2HA+bKUcF0x2UorF8qoOwgzStiQo8RD0y7W5zf0XsVfVYPKWBzdnS0QqeKedvCotDDYL04aHIfzUkrzQa/5zjeVivAquJCv67bGYa29s4pfTL2oq4p5x4U2pBXnuLz4c44DwimTZyTSjjwXsIW7mlp04=:i3nswYgQOGg3i8zMNvdyfw==",
      "MEDIAFIRE": "https://getcomics.info/links.php//LixVxF01adG/cANpunfH4iUMZSxYkvwg4V7EHDzy7PnO5I9CWUF+8gVeYeJJ/o+SV/pKuXenHUTCR/LF2edY3+brXVk5Wq5HlnIOTazzONnDSGJCU0gz7s4e8i3bfHF+lQ0EJIDVyKQfEXQJNQ0h/EQqCgVOySCvVyWV+TlThJufUrOXk87TjJX/+eIcqXa:1U3EVjAIuK3UT/abDAuD3g==",
      "MEGA": "https://getcomics.info/links.php/ptMxVJBcFAolkwWOmlaz8D+E/5DipU7Y5QeTXp1tSPxU1eJ5c9JvMBYBNm5dLCk02iIavFVZKPPKIha2BL5ZYvi0Tljm2CpjjSYqjQOO6ZQ=:r3WKF0oqkXe2h3G4hc1Oxg==",
      "ReadOnline": "https://readcomicsonline.ru/comic/i-am-batman-2021/14",
      "UFILE": "https://getcomics.ufile.io/zx9b1iib",
      "ZIPPYSHARE": "https://www21.zippyshare.com/v/61t3nJuy/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "37 MB",
      "Year": "2022",
    },
    "title": "I Am Batman #14 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-The-Knight-10-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Bruce Wayne’s worldwide journey has reached its conclusion–is he ready for the brutal final exam? The acclaimed series comes to a shocking, climactic end as this young Dark Knight will face the ultimate test!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/E0RUFg9Ph15GLawoXdq+upk+qIxKQ6xfQTcLWxRe+h4BHn6ay6ZfVio3nao+jsVaB9HYRRs1pfB94TGokAFmfR19F3VJr4dpjIAxKtdbvw9vRSEikFZGLiSru6qxpMb7YG9mIzKnfVHtRDZHzCBr3Kv3GogP/DjMJkA/8ri7TlhdG+MiKNCEK+YyDs7FUQCHxV2JFQXwWBSzSUjBbX6I8w==:64SnNCPvsmHV4VoJLFmUdA==",
      "MEDIAFIRE": "https://getcomics.info/links.php/zgQapzjYALv29uptouMRXhAvcRFTndgjy6Fn0aiDnJ1kpH4g09kmiQ61oHKvvLD5B7hdy3e8SwsH7gcoiTQ6iplGSWxOi6xkWIwupRmtL5M6zdEPwOmqpGhEzBUb9UXu9Idht1r8GzXQGU/HqbVhrG893C2U5xQBqk9kbngXkqsfOvSUTJh/i9az+Hx/wcn76qBKOmnDeGOP3fSmt6omccsgl6R37Wm+ED09f5yqUwg=:NCZ0SmTselQdeJ3ajNypXw==",
      "MEGA": "https://getcomics.info/links.php/64eETC/pVofAld+qFGnBVlBHa6/lIyOi2ZIiK5SuPkhUAxfDBgok7oKI217EEKT/pwViXc+9O9tMSaeh9WcoUPCw34kF+YSTts5oixm84Q4=:jPBfOGpJJzBxidOAOv4Kmg==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-the-knight-2022/10",
      "UFILE": "https://getcomics.ufile.io/64qe34sd",
      "ZIPPYSHARE": "https://www25.zippyshare.com/v/mLfRZWjd/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "38 MB",
      "Year": "2022",
    },
    "title": "Batman – The Knight #10 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Superman-Worlds-Finest-8-2022.jpg?fit=400%2C615&ssl=1",
    "description": "“RECKLESS YOUTH” continues as Superman and Batman struggle to mentor the new Boy Thunder! The Key has drawn the World’s Finest team into an impossible dilemma unlike any they’ve ever faced when the entire city of Metropolis goes into a deadly lockdown!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/k7E6EfgPpbhWMLIn/khYaaJlcOSoLRKs32lpkDplJKa5SCx113p8NWzOS+YKAA/qgSpN1gOvVIkIWDwpHgmZcIqpbJ0l5IZeTacJDu5jvvcb/n41Zr5I2zfHkk0+puffVi5nfRXVAXcyX18nw+zKw9vpSHCaHWXxqQxYyyIai5sDVvgiFJkqGHefFE3bJ8lmiEJmUBkJZ5gYKzBIHw1PlQ==:0S94KScYC3zAuR+O8fIi1w==",
      "MEDIAFIRE": "https://getcomics.info/links.php/5i6MI8quDWJZ40uk9iQydHPkziDX5/ozYqRXU82s5RnTsEWGBkje7T6S3ZjDyobEaSqDnsGnNYAfz+sQoRyYe+zOenlAQ9WRrouwC4zq2zQpCr+SVqqtdFqMk6zYJBdLcZ7RYZcq/uuN9ET8hs68aKQltO044a0MC8JPLKZRhsNNC/nryyP9PQxlt1lsU4odkS50JiMRP/XJ7zKnLABPeg==:f5rWS4VoozvW5L/9LQDgiw==",
      "MEGA": "https://getcomics.info/links.php/6rtAFEOUH7V1imGiLHOXEvSgF1dRKU3G9L1yNYSC9vHM+Ds2YwRUaj5GracQBmxtVADlu0qNuAPfSgW0WnfXPxUxNeKWB8VjtBFxIl8eCsg=:Q4F2pSdpoU/M86V3I7SyRA==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-superman-worlds-finest-2022/8",
      "UFILE": "https://getcomics.ufile.io/ctf2jlxg",
      "ZIPPYSHARE": "https://www4.zippyshare.com/v/jJtNoprR/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "38 MB",
      "Year": "2022",
    },
    "title": "Batman – Superman – World’s Finest #8 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-One-Bad-Day-The-Penguin-1-2022.jpg?fit=400%2C615&ssl=1",
    "description": "The Penguin’s criminal enterprise and the Iceberg Lounge have been stolen from underneath him by his former associate the Umbrella Man. The Umbrella Man has removed all of the rules for crime in Gotham City that the Penguin put in place, and the city is in chaos. The Penguin is a broken man, and he’ll have to travel through the burning streets of Gotham with a gun and a single bullet putting together a new crew to take back what he’s built. Will Batman help the devil he knows or face the devil he doesn’t in the form of the Umbrella Man? A crime epic from the team behind the critically acclaimed Other History of the DC Universe, John Ridley and Giuseppe Camuncoli–don’t miss it!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/ndrDsBHANfO+NhOHafwgk74ADx3ZSLi+xS1Dp//RaKSeJJPa1ON4KjeSqD/zE9aOK+TibodelPPw+3E4T0KQEPoV3CDIOPbSAG70CqARuER11qfayWNEH65wzU3QpBFGUlhM6Nh1xAtt6UPT6e6BYFqFHqI1IqLY9cPBUSLM7GcFpIbPuhdU0ZE9ZRhVlgp0GElABQwL7dFcvxMnFpSLvg==:dkJKbsMKjbWM2vBHpZMttw==",
      "MEDIAFIRE": "https://getcomics.info/links.php/9mFAC8bU3E4bx1WVnxrZmsjdp5I7BVfq8NB9yhcQqfzAlBheDJzSj/p+A5MWY8U3O3IcdAw1k6VNrkaX4t6/XEhQKxNT9AcjPUDt0Dx9hoKl5vw7RLUA1UTmfiVwvExQTBqH8FgoGnisbgxS6SJN9X6hdNFHBIy52o2g3VsA2Uw+c+BHuGgTm0ummYVebSZ9DHq4FYhqDgCIc38mP10sfg==:rKP5zHKhhYfFvTl0ene71g==",
      "MEGA": "https://getcomics.info/links.php/D1jaxig0qtUCnTYJ4PPJXnU5hQg2A+xzRXUeeW4iyyXoHXi0iOc4LsFKMCrMv2Z5Jdv3F/CDVNhq53mnValX08CmTala1XSOg90rrQeBmbY=:3oHKQo+Jk3exiKkfj8SQ4A==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-one-bad-day-the-penguin-2022/1",
      "UFILE": "https://getcomics.ufile.io/rlol5hk9",
      "ZIPPYSHARE": "https://www81.zippyshare.com/v/ipBmWVp0/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "102 MB",
      "Year": "2022",
    },
    "title": "Batman – One Bad Day – The Penguin #1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-Vol.-5-Fear-State-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Gotham City is on the brink of martial law as Batman fights a two-front battle! The Scarecrow is unleashing a devasting attack on the city while the Magistrate have made their move to invade! The BATMAN: THE JOKER WAR creative team of James Tynion IV and Jorge Jimenez continue their epic run with Gotham City on the brink of a violent evolutionary path and the danger level to Batman and his allies reaching a fever pitch! The Magistrate now have full support from the mayor to handle high-level law enforcement in Gotham and have branded the Dark Knight as Public Enemy #1. Batman must also contend with the return of the Scarecrow who pursues his own terrifying agenda as he takes this chaotic moment to launch his FEAR STATE. Includes Batman #112-117.",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/xN2V22VyHsqTsQoI+8f349juHrUQSkoaeltpWyhWXy2IQ2CUe1R97Bm3ylh8sZzC//d0wyBomV8ejOjidXU86kDD4udfeOvgHt0jQ3wLWovmGCMRcIGCRs++wXOVBHKQ4GYAusWdRTPnLqtPoPiCu6b2zZTE8nNPFnRw6Eh/FR6Gm4dkdS4eo96j+DN7f5Rgy0F094HB4ElxJHbsn1IJmA==:c6l8HnfW5rU/2lqgV9ayyQ==",
      "MEDIAFIRE": "https://getcomics.info/links.php/566YkLWV1DAyMljVKDJgRwFf3wNgfaxqXyhS77/CE+DIO/OySXdrV1M8TYAn8I1Idmd8BfFWw9An6Y/N8VWeND0H0G0I+G3Lle2uM/pztGqM9rOpAhlD1OjSi/b9nSQ7gDJ99ZP3KP52wghLt7YLYx8ITZZmxEqhoTQWn8ivgURk7D75fAju1mlg3xcB0xPW:Gpe8FHKNBHXMuxDZBPhkGA==",
      "MEGA": "https://getcomics.info/links.php/WcesdBdxuc8aH/8IgJ+n2dlQ7mGEQzZ+UEfAXD0yfDH0dvwFedW6lYQKV+L3nskhhFaOLFNjM6rmHxxXIkAb+OywekU+cJDaZwRpUL3aFUo=:E0PJi5DIKKxMMzqciRAAMA==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-2016",
      "UFILE": "https://getcomics.ufile.io/ki4lzez0",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "699 MB",
      "Year": "2022",
    },
    "title": "Batman Vol. 5 – Fear State (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-by-Doug-Moench-Kelley-Jones-Vol.-1-2014-Fan-Made-TPB.jpg?fit=400%2C608&ssl=1",
    "description": "In these moody 1990s tales, Batman emerges from the shadows to battle corruption in Gotham City, and to face foes including Killer Croc, The Scarecrow, Mr. Freeze, Two-Face and more. These tales by the classic Batman creative team of writer Doug Moench and artist Kelley Jones include guest-appearances by Nightwing, Robin, Deadman and Swamp Thing. Collects BATMAN #515-525, 527-532 and 535.",
    "downloadLinks": {
      "MEDIAFIRE": "https://getcomics.info/links.php/roiPsdBQwVrA1Z8ud6w3rjs4Gsp4nRxhURagXKTLe0locTGWPtulm6o+dIsnf4D/82C03vvH6op9FxInmNu4aZgU2ocQzT7D21qh67DiMZEWpMmmO7LpR8EobASPpyXadcEO7aYQglvh+fI3k09eu+lQ7dO2hHypqQOh4Yf3/7w=:REM8p/J/x3vTQWINKMzOcQ==",
      "MEGA": "https://getcomics.info/links.php/xtTLGZ7QMLU+7YVh32MmjGssaHGC6Lp2XIntExX+ILZ9IX3rARTNtJ+tqzA6gBwcmqomB2MmXPHNqnj9aMLtopAnT/Sfe1JMOeyA3MMankE=:SDTcfAj6k3nL4KVnPVgBwA==",
      "UFILE": "https://getcomics.ufile.io/a7z52nn4",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "828 MB",
      "Year": "2014",
    },
    "title": "Batman by Doug Moench & Kelley Jones Vol. 1 (2014) (Fan-Made TPB)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/The-Batman-and-Scooby-Doo-Mysteries-1-2022.jpg?fit=400%2C615&ssl=1",
    "description": "THE TEAM-UP SERIES YOU DEMANDED IS BACK! An odd thriller pulls Batman, Robin, the Batmen of All Nations, Scooby, and the gang right into a jet-setting race to among the most haunted locations on Earth. Even scarier, the path leads straight to Ra’s al Ghul and his League of Assassins!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/3MOd8JvewawpMiO8SUfWQ5XSERxNYH/kNzL5MFw3jS7JxB5Ic5mNlu6UHl6AqsP6AbTRXwoRCZDpbYRhkiRXdY82ZU6p8IL3utu2hO9EqdOKVmzhaH+e3i+Ri0VBxrP87dojaO6xt8+KUBLGlsr2zfXzBfyicV5I4GAC7nB/pDKzaRUSge81Cmbg9CxZvuCkyp0+9J30dXTCNq4xX9fHup8n6a/zFWmqJjZAZYN5rFs=:gDeWHcDAs+1EEndN3ZY++g==",
      "MEDIAFIRE": "https://getcomics.info/links.php/exTam+PoGDAoHb/SkDuyfufXNGDa0y+Jb1dk2xAndbLDlXAbmgYwK5AQaVxwNAQ/0xA8sjDz40+OC6am+tJrUVShFT5nAsVNX2w2NthR7G7MxpMDWGWm8bcLVzTb99NgI9fInS9hQfohdixIj9YVDOtYYT5cY4qKe9EOVfwM1PU7GUOELyR4y8z7GZDo/DR0ccVI1M4EaeC7jAZHx5e4nHRuRZzAy5lMn9087Qll0E2NyefYbStHCUiUQ46igv0Q:dbqJ8BX5+83alWDtIAxnSw==",
      "MEGA": "https://getcomics.info/links.php/F2zqVYWHCtuWFvOwEcswOI/dQq51N69ACtu5rMKS1xJqTmDyMR+M0V77hPCKxUEl/d/VSepjRP7g6N4qyp2MfGOJrjrTvd5k8hPXPi7gKU0=:A7scDs9vYTTrGXfJw6pHuw==",
      "ReadOnline": "https://readcomicsonline.ru/comic/the-batman-and-scoobydoo-mysteries-2022/1",
      "UFILE": "https://getcomics.ufile.io/d3lxia7q",
      "ZIPPYSHARE": "https://www118.zippyshare.com/v/FLGHbyKH/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "106 MB",
      "Year": "2022",
    },
    "title": "The Batman and Scooby-Doo Mysteries #1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Batman-vs.-Robin-2-2022.jpg?fit=400%2C615&ssl=1",
    "description": "HOUSE OF SECRETS…HOUSE OF DEATH. In the wake of Damian Wayne’s devastating attack on the Batcave, Bruce and Alfred are on the run and running out of time! Magic users across the planet are experiencing dangerous and deadly power flares, and Batman must solve this mystery before his friends and allies are turned to ash. To crack this case, the Dark Knight is going to need the help of one of the greatest masters of the magic arts on the planet…Zatanna! What awaits our ragtag group of heroes in the House of Secrets? Can Damian break free of Nezha’s possession spell before he murders his own father? All this and more in the explosive second chapter of Batman vs. Robin!",
    "downloadLinks": {
      "DownloadNow": "https://getcomics.info/links.php/1gsoCsGzfyTnuqIVD8rwf6mqcjC/7tnlBjEBsCfBic5GTvGAk1bcATfZxvvlJXUx781H5yyVDnHPLjin9CmHLIj+7pNOPCSRkzosRVAbMGfmi4S23uMPQ559bhLVILFyZXGZI/nOOhmVJKk7IR9CP1xb0GXQ0qx2N7rxpcLmQItaplGo6La2VMpfXChkiH9V:sPDOFqC7ecQsKfYbowK+Vg==",
      "MEDIAFIRE": "https://getcomics.info/links.php/8jA8LCa6TYa/qEXfK1l/RkIAo1mxWdFzDKXl9c6KeqzZ1ohpa1Yhlo2clwecp0T2lB6XS8jijjwSPyJRzM5VPmX8iKjvCcuFdkTDa6twfcF2p14tfDxp4Kvpw3iS35vE3FKymk/uYqnCAx/8NYM5LwdJAENxvEmme73yOCuGVaociZXvx3keS36JonW37rvE:PA+qSUIJo+waJrbcBCThFg==",
      "MEGA": "https://getcomics.info/links.php/85XTkeYL133qwpVrA3o2dJq4REyHSQjdlWX9O5ajL8IXR1nPetI/iU3SkjVEH5E6FbQEfVG+RmYbnIbAK2BhqEsNmfheRUvu2vSQHnulwsc=:TWQCNV41iRg499mGkFIpYw==",
      "ReadOnline": "https://readcomicsonline.ru/comic/batman-vs-robin-2022/2",
      "UFILE": "https://getcomics.ufile.io/77brzepw",
      "ZIPPYSHARE": "https://www100.zippyshare.com/v/QoSkunzE/file.html",
    },
    "information": {
      "ImageFormat": "JPG",
      "Size": "62 MB",
      "Year": "2022",
    },
    "title": "Batman vs. Robin #2 (2022)",
  },
]
`;
